src/dashboards/components/AutoRefreshInput.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {FC} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Components
+import DurationInput from 'src/shared/components/DurationInput'
+import {ButtonGroup} from '@influxdata/clockface'
+// Actions
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+// Types
+import {AutoRefreshStatus} from 'src/types'
+
+// Actions
+import {
+  setAutoRefreshInterval,
+  setAutoRefreshStatus,
+  resetDashboardAutoRefresh,
+  setAutoRefreshInputValue,
+} from 'src/shared/actions/autoRefresh'
+
+// Selectors
+import {getCurrentDashboardId} from 'src/dashboards/selectors'
+import {getAutoRefreshForDashboard} from 'src/shared/selectors/autoRefresh'
+
+const SUGGESTIONS = [
+  'None',
+  '10s',
+  '15s',
+  '30s',
+  '1m',
+  '5m',
+  '10m',
+  '30m',
+  '1h',
+]
+
+const AutoRefreshInput: FC = () => {
+  const dispatch = useDispatch()
+  const currentDashboardId = useSelector(getCurrentDashboardId)
+
+  const autoRefresh = useSelector(getAutoRefreshForDashboard)
+  const handleChooseAutoRefresh = (selection: string) => {
+    if (selection === 'None') {
+      dispatch(resetDashboardAutoRefresh(currentDashboardId))
+      return
+    }
+    const unit = selection[selection.length - 1]
```

Comment:
I have no doubt this work, but it's a little confusing for me since it's making me think that `selection` is an `object` or an `array` based on the method used. 

nit: `string.charAt(selection.length - 1)`